libc/src/ctype/isalpha.cpp

```diff
@@ -14,8 +14,6 @@
 
 namespace LIBC_NAMESPACE_DECL {
 
-// TODO: Currently restricted to default locale.
-// These should be extended using locale information.
 LLVM_LIBC_FUNCTION(int, isalpha, (int c)) {
   return static_cast<int>(internal::isalpha(static_cast<unsigned>(c)));
```

Comment:
how about adding assertions to check if the local is default? Would that require too much work?

Reply:
Do we export the libc assertions? I thought those were only internal.